pkgs/by-name/se/seagoat/package.nix

```diff
@@ -0,0 +1,53 @@
+{lib, fetchPypi, python3, python3Packages, ripgrep}:
+
+python3Packages.buildPythonPackage rec {
+  pname = "seagoat";
+  version = "0.48.4";
+  pyproject = true;
+
+  propagatedBuildInputs = with python3.pkgs; [
+    poetry-core
+    appdirs
+    blessed
+    chardet
+    flask
+    deepmerge
+    chromadb
+    gitpython
+    jsonschema
+    pygments
+    requests
+    nest-asyncio
+    waitress
+    psutil
+    stop-words
+  ];
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-MnRUl6hDfl1NGcV2cXZO+WxVqWUD4TmhP+KkMEYVve8=";
+  };
+
+  stop-words = python3Packages.buildPythonApplication rec {
```

Comment:
Better package this separately in python-modules

Reply:
I'm hesitant to package this since it's just a python wrapper around a bunch of stop words in multiple languages. Kinda like seclists.

Reply:
Apologies for necrobumping but would you still require me to package `stop-words` separately?

Reply:
Better to, because it will be built in either ways, and exposing it is better. Also `meta` field is always required no matter whether it's only passed to another derivation or not, as it will be consumed by hydra.